app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/MessageComposer.kt

```diff
@@ -204,13 +204,13 @@ private fun MessageComposer(
                                 )
                             }
                             .background(color = colorsScheme().backgroundVariant)
-                            .padding(bottom = if (isConversationMember) 0.dp else dimensions().spacing16x)
+                            .padding(bottom = if (isSendingMessagesAllowed) 0.dp else dimensions().spacing16x)
```

Comment:
I would rather not draw the `messageInput` if `!isSendingMessagesAllowed` instead of adding some extra padding. 
It feels a bit like a "tweak". I wonder if it would still be hidden if someone changes later on the height of the input or does any other modification.